### PR TITLE
changed order of menu

### PR DESCRIPTION
**Pull request by numbap**
## [ADO-107286](https://dev.azure.com/VP-BD/DECD/_workitems/edit/XXX)

fixed [AB#107286](https://dev.azure.com/VP-BD/d5ca3cd4-19cb-4c8c-b3e6-a7068f4677e4/_workitems/edit/107286)

### Description

List of proposed changes:

- New menu order should be 1) dashboard, 2) profile, 3) security settings and 4) sign out
-

### CODE REVIEW

File: components/ExitBetaModal.js [hunk 1]
```diff
@@ -16,7 +18,7 @@ export default function ExitBetaModal(props) {
           role="heading"
           aria-level="1"
         >
-          {props.popupTitle}
+          {sentenceCase(props.popupTitle)}
```
Comment (krischarbonneau) [inline components/ExitBetaModal.js:21]: I'm not sure this is necessary. Given we are in control of our own content here, we can just change the casing in AEM

File: components/ExitBetaModal.js [hunk 2]
```diff
@@ -2,8 +2,10 @@ import { solid } from '@fortawesome/fontawesome-svg-core/import.macro'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { Button } from '@dts-stn/service-canada-design-system'
 import PropTypes from 'prop-types'
+import { sentenceCase } from '../lib/text'
 
 export default function ExitBetaModal(props) {
+  console.log(sentenceCase('this is my Sentence to casify.'))
```
Comment (krischarbonneau) [inline components/ExitBetaModal.js:8]: Please remove this :)

File: .env copy.local [hunk 3]
```diff
@@ -0,0 +1,19 @@
+NEXT_PUBLIC_BUILD_DATE = 12345
```
Comment (krischarbonneau) [inline .env copy.local:1]: Was this an accidental commit?

**Review by krischarbonneau — CHANGES_REQUESTED** [review]
The changes to the menu order look good. I left some comments about the other changes.

File: components/ExitBetaModal.js (hunk 1 above)
Comment (shewood) [reply components/ExitBetaModal.js:21]: I agree. This looks to be a change that content would make and not a dev change.

**Review by shewood — CHANGES_REQUESTED** [review]
Please don't commit the env file.

File: .env copy.local (hunk 3 above)
Comment (numbap) [reply .env copy.local]: Fixed.

File: components/ExitBetaModal.js (hunk 2 above)
Comment (numbap) [reply components/ExitBetaModal.js]: fixed

File: components/ExitBetaModal.js (hunk 1 above)
Comment (numbap) [reply components/ExitBetaModal.js:20]: Fixed.

**Review by krischarbonneau — APPROVED** [review]
Looks good

**Review by shewood — APPROVED** [review]
Looks good!

**Pull request merged**